src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-noninteractive-element-interactions
+            <li
+              className={classNames(
+                { 'is-active': tab.id === selectedTab.id },
+              )}
+              data-cy="Tab"
+              key={tab.id}
+              onClick={() => {
```

Comment:
The `onClick` is being defined within the render method which will create a new function every time the component is rendered. It's better to define this function outside of the render method to avoid unnecessary re-renders and improve performance.